src/devices/video/bt459.cpp

```diff
@@ -115,26 +140,37 @@ void bt459_device::set_component(rgb_t *arr, int index, u8 data)
 	}
 }
 
-READ8_MEMBER(bt459_device::read)
+READ8_MEMBER(bt459_device::address_lo_r)
 {
-	u8 result = 0;
+	// reset component pointer and return address register lsb
+	m_address_rgb = 0;
```

Comment:
You need to guard the read side effect with `if (!machine().side_effect_disabled())`